src/Payum/Paypal/ExpressCheckout/Nvp/Api.php

```diff
@@ -426,6 +426,24 @@ public function doExpressCheckoutPayment(array $fields)
      *
      * @return array
      */
+    public function cancelRecurringPaymentsProfile(array $fields)
```

Comment:
there is no need for this API method. use `manageRecurringPaymentsProfileStatus` one